src/api/get-installations.js

```diff
@@ -103,10 +103,13 @@ async function findWindowsBrowsers () {
     var machineRegisteredBrowsers = await searchInRegistry('HKEY_LOCAL_MACHINE');
     var userRegisteredBrowsers    = await searchInRegistry('HKEY_CURRENT_USER');
     var installations             = Object.assign(machineRegisteredBrowsers, userRegisteredBrowsers);
-    var edgeAlias                 = await detectMicrosoftEdge();
 
-    if (edgeAlias)
-        installations['edge'] = edgeAlias;
+    if (!installations['edge']) {
```

Comment:
Maybe we can just a detected legacy Edge to the installations list? I don't think that it will hurt someone.
```js
const legacyEdge = await detectMicrosoftEdgeLegacy();

if (legacyEdge)
    installations['legacy-edge']=legacyEdge;

if (legacyEdge && !installations['edge'])
    installations['edge'] = legacyEdge;
``` 
```